packages/loader/src/spinner.scss

```
@@ -1,5 +1,19 @@
 @import '@axa-fr/react-toolkit-core/src/common/scss/core.scss';
 
+.af-loader {
+  min-height: 94px;
+  position: relative;
+
+  .af-loader-on {
```

tu peux utiliser le '&' 
```suggestion
  &-on {
```